.github/workflows/CodeQC.yaml

```diff
@@ -6,19 +6,19 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-    
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v1
-      with:
-        python-version: 3.8
-    
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install pylint
-    
-    - name: Analysing the code with pylint
-      run: |
-        pylint *.py;
\ No newline at end of file
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pylint
+
+      - name: Analysing the code with pylint
+        run: |
+          find . -type f -not -path "./venv/*" -name "*.py" | xargs pylint
```

Comment:
Should we expect this call to pylint to handle linting all the files at once , or does it run the linter on each individual file in sequence?

Reply:
@christinaholtNOAA attached sample output.

It will lint each file consecutively.
[pylint_sample_output.txt](https://github.com/ufs-community/workflow-tools/files/8491396/pylint_sample_output.txt)

Reply:
We should probably be linting the whole package together. It will catch things we may want to be aware of at a package level like duplicate code and such.

Reply:
@christinaholtNOAA in UW-44 branch, since packages were introduced here, you would want to call ```pylint src```.

![image](https://user-images.githubusercontent.com/69773028/163456848-9c8ec371-bb29-4443-a64c-11f056337ca2.png)